docnado/style/templates/base.html

```diff
@@ -93,8 +93,8 @@
                 </li>
                 {% else %}
                 <li>
-                    <input type="checkbox" {% if nav is defined and key not in nav[0] %}checked{% endif %} id="#nav-{{ key }}">
-                    <label for="#nav-{{ key }}">
+                    <input type="checkbox" {% if nav is defined and key not in nav[0] %}checked{% endif %} id="#nav-{{key|replace(' ', '-')}}">
+                    <label for="#nav-{{key|replace(' ', '-')}}">

```

Comment:
Rather than putting this logic in the template, would it be better to instead put it in the `NavTree` logic: https://github.com/HEInventions/docnado/blob/master/docnado/navtree.py#L33

Reply:
It doesn't work without a big refactor. That way also replaces spaces in the link text with dashes which isn't good from a UX perspective. The way round it is to have a "key" and a "key with dashes" element in the nav item object... and that way would require changing the NavTree logic AND the base template anyway.

Reply:
@wilad was this updated with the changes to `NavTree` we looked at yesterday?

Reply:
Yes